lib/routes/account.js

```diff
@@ -213,10 +215,17 @@ module.exports = function (
             )
         }
 
+        function optionallyOverrideCreatedAt () {
+          var createdAt = parseInt(query._createdAt)
+          if (createdAt < Date.now() && process.env.NODE_ENV !== 'production') {
```

Comment:
There's a `config.isProduction` flag that can be used for this purpose (and whose definition leads me to believe that the appropriate `NODE_ENV` string is `"prod"`)